src/theme.ts

```diff
@@ -6,3 +6,21 @@ export const ukriGreen = "rgba(0, 130, 118, 1)";
 export const Title = styled(H1)`
     color: ${ukriGreen};
 `;
+
+export const SettingsListItem = styled.div`
+    width: 100%;
+    height: 60px;
+    background-color: #d7d7d7;
+    line-height: 60px;
+    padding: 0 15px;
+    margin-bottom: 30px;
+`;
+
+export const FauxLink = styled.button`
```

Comment:
You'll need to add a font here, and fix the vertical alignment

Reply:
I've made the changes mentioned, hoping that's the vertical alignment sorted as mentioned.